### ISSUE TITLE
Web site with documentation is not accessible 

### ISSUE THREAD

**Issue by ikhvostenkov** [OpenZWave/open-zwave#2644]
http://www.openzwave.com site referenced in the README is not working. Did you move to the other host location? This would be helpful if this would be possible to access extended documentation.

**Comment by BilalAzhar0** [comment]
Website is still not accessible....

**Comment by blhoward2** [comment]
That’s because the project is all but dead after the original maintainer left. The website has been down for months. I’d suggest you give zwavejs a look as a replacement.

**Comment by nechry** [comment]
@blhoward2 I am not the owner of the openzwave.org website and I have no admin access to the site. From my side I can not do anything to correct.

On the other hand, openzwave doesn't have maintainers either.

**Comment by blhoward2** [comment]
Yes, I knew you didn’t own the domain. Sorry, my post was confusing. I thought you were maintaining this project now, though, so I meant an update about the ozw project altogether (which I see you posted).

**Comment by nechry** [comment]
Yes I have participated for a very long time, mainly in the quality control of configuration files. 
Without asking, I inherited the project, but I don't have the C++ skills required to follow the required changes.
So yes on paper, I'm the maintainer, but in fact it's something else...

I think openzwave will have been a great adventure that will even have helped to democratize Z-Wave to a greater mass of users.

Most of the home automation solutions are passed on Z-Wave JS all like me. Personally, I will invest my energies in Z-Wave JS.

If there are contributors interested and willing to continue openzwave, the door is open as in the past.
